.github/workflows/CI-pr.yml

```diff
@@ -257,8 +263,6 @@ jobs:
       #     crate: cargo-nextest
 
       - uses: Swatinem/rust-cache@v2.7.0
```

Comment:
Let's use `save-if: false` in both jobs (stable and MSRV toolchain tests), let's not push to cache from PRs, but only pull from cache.